app.js

```diff
@@ -9,3 +11,19 @@ const signUpPassword = document.getElementById('sign-up-password');
 // Wire up sign in and sign up forms to supabase
 // Redirect to /other-page on successful auth
 // Redirect to /other-page when page loads if user is authenticated
+
+redirectIfLoggedIn();
+
+signUpForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    console.log('sign up', signUpEmail.value, signUpPassword.value);
+    await signupUser(signUpEmail.value, signUpPassword.value);
+    redirectIfLoggedIn();
+});
+
+signInForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    console.log('sign in', signInEmail.value, signInPassword.value);
```

Comment:
Great Job! One thing I would change is add an eslint disable console.log for the files you want to leave them in so you don't get all of the warnings